charts/kong/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased
 
 * Added support for ServiceMonitor relabelings allowing labels manipulation before scraping.
+* Added `ingressController.konnect.controlPlaneID` and removed `ingressController.konnect.runtimeGroupID`
```

Comment:
Should we notify users that this is a breaking change and tell them to use `controlPlaneID` instead of `runtimeGroupID`?

Reply:
Yes. It's a breaking change.

Apart from indicating in the GitHub Release page, do you have any other suggestions?

Reply:
Thank you! Updated.